packages/core/src/utils.ts

```diff
@@ -40,7 +42,9 @@ export function block({
 	return () => {
 		input.off('keypress', clear);
```

Comment:
I would suggest to rename the const `clear` to `sigintOrClear` so it reads that this functions does more than just clearing the screen